configs/dlstreamer/framework-pipelines/stream_density.sh

```diff
@@ -201,7 +211,7 @@ do
 			echo "Warning: No FPS returned from pipeline$i.log"
 			STREAM_FPS_LIST=`tail -20 /tmp/results/pipeline$i.log`
 		fi
-		echo "DEBUG: $STREAM_FPS_LIST" >> $log
+		#echo "DEBUG: $STREAM_FPS_LIST" >> $log
```

Comment:
remove or uncomment

Reply:
i'll remove since the stream_density.log will be too noisy if uncomment